pom.xml

```diff
@@ -372,6 +373,12 @@
                 <version>${parquet-avro.version}</version>
             </dependency>
 
+            <dependency>
```

Comment:
It seems `parquet-avro` is dependent on `parquet-jackson` in runtime scope after pumping up to 1.15.1; this is the reason why we need it here.

Reply:
Actually, parquet-avro 1.15.1 contains parquet-jackson 1.15.1 which lead fail to complie the project with JDK8
I replace paruqet-jackson 1.15.1 with 1.13.1 to build the project correctly.

Reply:
Got it, we may remove this after upgrading the JDK version, maybe add a comment to add this TODO.